php-frontend/src/test/java/org/sonar/php/symbols/FunctionSymbolIndexTest.java

```diff
@@ -87,7 +86,7 @@ private FunctionSymbolIndex index(FunctionSymbolData... data) {
     for (FunctionSymbolData f : data) {
       projectSymbolData.add(f);
     }
-    return FunctionSymbolIndex.create(new HashSet<>(Arrays.asList(data)), projectSymbolData);
+    return FunctionSymbolIndex.create(new ArrayList<>(Arrays.asList(data)), projectSymbolData);
```

Comment:
```suggestion
    return FunctionSymbolIndex.create(Arrays.asList(data), projectSymbolData);
```
The list can be immutable as it will not be changed afterward.